content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md

```diff
@@ -0,0 +1,48 @@
+---
+weight: 60
+title: Release 10.17.0
+layout: redirect
+---
+
+The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for the Cumulocity IoT Platform 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
+
+Download the Cumulocity IoT Edge VM based distribution version 10.17 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+## Cumulocity IoT Edge Appliance VM
+
+### Mongo DB update
+
+MongoDB has been updated to version 5.0.14.
+
+### Known issues
+
+|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
+|:---|:---|:---
+|CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** To Disable and enable the microservice hosting feature again. For further information see [Enabling the microservice hosting feature](https://cumulocity.com/guides/edge/configuration/#configuring-microservices).
+|CIE-4376|UI|**Issue:** When switching to a language other than English, the translations for the security menu and security page are not available.
+
+For new enhancements in Cumulocity IoT, see [What's new](/release-10-17-0/whatsnew-10-17-0/).
```

Comment:
There is a similar statement above linking to the youtube video. Perhaps link to both places above?

Reply:
Added this link along with YouTube link.